docs/asciidoc/static/upgrading.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[upgrading-logstash]]
+== Upgrading Logstash
+
+[IMPORTANT]
+===========================================
+Before upgrading Logstash:
+
+* Consult the <<breaking-changes,breaking changes>> docs.
+* Test upgrades in a development environment before upgrading your production cluster.
+===========================================
+
+=== Upgrading Using Package Managers
+
+This procedure uses <<package-repositories,package managers>> to upgrade Logstash.
+
+1. Shut down your Logstash pipeline.
```

Comment:
Maybe add "Take care to also shutdown inputs that are sending events to Logstash"